examples/gallery/maps/tilemaps.py

```diff
@@ -37,8 +37,8 @@
 fig.tilemap(
     region=[-157.84, -157.8, 21.255, 21.285],
     projection="M12c",
-    # Use the Stamen.Watercolor option from contextily
-    source=contextily.providers.Stamen.Watercolor,
+    # Use the U.S. Geological Survey  Imagery Topo web tiles from contextily
+    source=contextily.providers.USGS.USImageryTopo,
```

Comment:
The USImageryTopo map looks like this:

![image](https://github.com/GenericMappingTools/pygmt/assets/23487320/38afb4c5-4435-4375-a0d6-bce7c1ac4b76)

Alternatively, I was also thinking that the [CartoDB basemaps](https://github.com/CartoDB/basemap-styles/tree/master#the-cartography) could be a good option (that is also global in coverage). E.g. CartoDB Positron:

![cartodb_positron](https://github.com/GenericMappingTools/pygmt/assets/23487320/0cf70ed7-1a29-4750-a77a-02da66e722a0)

Reply:
> I was also thinking that the [CartoDB basemaps](https://github.com/CartoDB/basemap-styles/tree/master#the-cartography) could be a good option (that is also global in coverage). E.g. CartoDB Positron which has a lighter colour and might be better to overlay other data on top:

Sounds good to me.

Reply:
Ok, switched to CartoDB Positron in 996f898a1316249dfa01a8979e7c02c588ed6413.